src/components/app/DetailsContainer.js

```diff
@@ -24,10 +25,11 @@ function dispatchResizeEvent() {
 
 type StateProps = {|
   +selectedTab: TabSlug,
+  +isSidebarOpen: boolean,
 |};
 
-function DetailsContainer({ selectedTab }: StateProps) {
-  const Sidebar = selectSidebar(selectedTab);
+function DetailsContainer({ selectedTab, isSidebarOpen }: StateProps) {
+  const Sidebar = isSidebarOpen && selectSidebar(selectedTab);
```

Comment:
Very small nit,
can we switch the order of `selectSidebar(selectedTab)`  and `isSidebarOpen` here?

1. `mapStateToProps` and the params both are in that order
2. As `isSidebarOpen` depends on `selectSidebar(selectedTab)`, it would fail first and be more fault tolerant.

Reply:
I don't understand this comment @zoepage. Especially `As isSidebarOpen depends on selectSidebar(selectedTab)`, what do you mean here ? I also don't understand the `fault tolerant` part.

The function `selectSidebar` returns the component I'll use as the Sidebar, I can't put it first otherwise I'll lose the result.
Moreover in a shortcircuit construct like `&&` it's better for performance to put the less costly operation first (in this case a boolean is less costly than a function call). Here the performance is not important because `selectSidebar` is very cheap too.